common/src/main/java/com/wynntils/features/overlays/TradeMarketBulkSellFeature.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.overlays;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.config.Category;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigCategory;
+import com.wynntils.core.config.RegisterConfig;
+import com.wynntils.core.features.Feature;
+import com.wynntils.handlers.chat.event.ChatMessageReceivedEvent;
+import com.wynntils.mc.event.ContainerSetSlotEvent;
+import com.wynntils.mc.event.ScreenOpenedEvent;
+import com.wynntils.screens.base.widgets.WynntilsButton;
+import com.wynntils.utils.mc.ComponentUtils;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.wynn.ContainerUtils;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.client.gui.screens.inventory.ContainerScreen;
+import net.minecraft.client.gui.screens.inventory.MenuAccess;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.inventory.ChestMenu;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@ConfigCategory(Category.OVERLAYS)
+public class TradeMarketBulkSellFeature extends Feature {
+
+    @RegisterConfig
+    public Config<Integer> bulkSell1Amount = new Config<>(64);
+
+    @RegisterConfig
+    public Config<Integer> bulkSell2Amount = new Config<>(1728);
+
+    @RegisterConfig
+    public Config<Integer> bulkSell3Amount = new Config<>(6399);
+
+    private static final Pattern ITEM_NAME_PATTERN =
+            Pattern.compile("§6Selling §f(\\d+|\\d+,\\d+) ([^À]*)À*§6 for §f[\\d,]*§7² Each");
+    private static final String SELL_DIALOGUE_TITLE = "What would you like to sell?";
+    private static final int SELLABLE_ITEM_SLOT = 10;
+    private static final int AMOUNT_ITEM_SLOT = 11;
+    private static final List<SellButton> sellButtons = new ArrayList<>();
+    private static final int BUTTON_WIDTH = 60;
+
+    private boolean shouldSend = false;
+    private int amountToSend = 0;
+
+    @SubscribeEvent
+    public void onSellDialogueOpened(ScreenOpenedEvent e) {
+        if (!(e.getScreen() instanceof ContainerScreen cs)) return;
+        if (!ComponentUtils.getUnformatted(cs.getTitle()).equals(SELL_DIALOGUE_TITLE)) return;
+
+        sellButtons.clear();
+        sellButtons.add(new SellButton(cs.leftPos - BUTTON_WIDTH, cs.topPos, 0, true));
+        sellButtons.add(new SellButton(cs.leftPos - BUTTON_WIDTH, cs.topPos + 21, bulkSell1Amount.get(), false));
```

Comment:
Maybe you can make additional buttons optional, so if the amount is 0 they are not drawn? And perhaps have the defaults 64, 0, 0, so there is the default of selling a stack, and selling all, and then people can tweak their config if they are prolific sellers?